src/bundle/Resources/public/js/alloyeditor/src/toolbars/config/base-table.js

```diff
@@ -24,4 +27,8 @@ export default class EzConfigTableBase {
     getConfigName() {
         return '';
     }
+
+    getArrowBoxClasses() {
```

Comment:
the `setPosition` and `getArrowBoxClasses` should be removed from the constructor as it is in the original PR

Reply:
I removed it early, but when rollback this file I forgot to remove it again